server/conf/application.conf

```diff
@@ -128,6 +128,24 @@ applicant_generic_oidc.first_name_attribute = ${?APPLICANT_OIDC_FIRST_NAME_ATTRI
 applicant_generic_oidc.middle_name_attribute = ${?APPLICANT_OIDC_MIDDLE_NAME_ATTRIBUTE}
 applicant_generic_oidc.last_name_attribute = ${?APPLICANT_OIDC_LAST_NAME_ATTRIBUTE}
 
+
+## Login.gov OIDC integration
+# https://developers.login.gov/oidc/
+login_gov.provider_name = "LoginGov"
+login_gov.client_id = "urn:gov:gsa:openidconnect.profiles:sp:sso:ar:civiform_dev"
+login_gov.client_id = ${?LOGIN_GOV_CLIENT_ID}
+login_gov.discovery_uri = "https://idp.int.identitysandbox.gov/.well-known/openid-configuration"
+login_gov.discovery_uri = ${?LOGIN_GOV_DISCOVERY_URI}
+
+# Additional Scopes should be space separated values.
+login_gov.additional_scopes = ${?LOGIN_GOV_ADDITIONAL_SCOPES}
```

Comment:
We don't have default values for this one? Or it will be empty by default?

Reply:
There's no need for a default additional scopes.